pages/admin/content-type-builder/[entity_type_slug].js

```diff
@@ -167,7 +167,12 @@ export default function ContentTypeBuilder({cache}) {
             </div>
     
             <div className="d-flex justify-content-between align-items-start mt-0 mx-0 px-0">
-              <AppCreatebutton title='Add another field' onClick={() => dispatch({type: SET_SHOW, payload: true})} link='' />
+              
+              <AppCreatebutton title="Add another field"
```

Comment:
Please rename `<AppCreatebutton />` to `<AppCreateButton/>`
`title="Add another field"` should be on the next line. It should look something like this:

```
<AppCreateButton
   title="Add another field"
   onClick={() => dispatch({type: SET_SHOW, payload: true})}
   link=''
/>
```

Reply:
Please fix the capitalization of the component. 

Convert `<AppCreatebutton />` to `<AppCreateButton />`